php/libraries/NDB_BVL_InstrumentStatus_ControlPanel.class.inc

```diff
@@ -105,9 +105,11 @@ class NDB_BVL_InstrumentStatus_ControlPanel extends NDB_BVL_InstrumentStatus
 
         if (empty($this->subtest)) {
             $hasAccess = $this->_hasAccess();
-
-            if ($this->getDataEntryStatus() == null && $hasAccess) {
-                $this->setDataEntryStatus('In Progress');
+            if ($hasAccess) {
```

Comment:
this still needs the `$this->getDataEntryStatus() == null`, because otherwise 1. the select isn't getting called and 2. it's updating the database unnecessarily.

Reply:
@driusan that makes sense. I added it again.